node/internal/actors/process_actor.go

```diff
@@ -37,49 +40,38 @@ type processActor struct {
 
 	retryCounter int
 
-	cancel  context.CancelFunc
-	logger  *slog.Logger
-	self    *goakt.PID
+	self *goakt.PID
+
 	process *OsProcess
+	stdout  logCapture
+	stderr  logCapture
 }
 
-func createNewProcessActor(
-	ctx context.Context,
-	logger *slog.Logger,
-	nc *nats.Conn,
-	processId string,
-	argv []string,
-	namespace string,
-	workloadType string,
-	processName string,
-	ref *ArtifactReference,
-	env map[string]string,
-	runType string,
-	triggerSub string,
-	retryCount int,
-) (*processActor, error) {
-
-	ret := processActor{
+func NewProcessActor(ctx context.Context, logger *slog.Logger, nc *nats.Conn, m *actorproto.StartWorkload, ref *ArtifactReference, clearEnv map[string]string) (*processActor, error) {
```

Comment:
Even though the package is internal, one nit is that this function is exported while the return value type of `processActor` is not. Suggest naming `newProcessActor`.

Reply:
[d8ffde8](https://github.com/synadia-io/nex/pull/471/commits/d8ffde819ea7a02069f4cf31b9b4058b8943cccc)